.github/workflows/docker-build.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           images: ${{ env.IMAGE }}
 
       - name: Login to Docker Hub
+        if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Only login with secrets on pushes with tags. Same logic as in line 50.

Reply:
Got it - thanks for the fix @tnightengale !